### ISSUE TITLE
[issue]: 

### ISSUE THREAD

**Issue by alastair1970** [ventoy/Ventoy#2446]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

Latest

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

Legacy BIOS Mode

### Partition Style

MBR

### Disk Capacity

2tb

### Disk Manufacturer

Sandisk

### Image file checksum (if applicable)

None

### What happened?

I’ve installed ventoy on a 2tb Sandisk ssd.   I reserved 2tb-34gb at the bottom.   When I use windows explorer to view the drive I only see one drive called ventoy.  Where are my 2tb-34gb?  Do I need to do something to see the spare space.     Also, when I try to install windows 10 pro to the usb it fails saying it can’t install to usb.  My plan was to install windows to USB so I could use my windows profile on any computer.

**Comment by wiz64** [comment]
check with your partition schemes, you can manage them later as well, I am not sure about the windows part, but ventoy by itself doesn't install any operating to your system, it boots iso files from disks, usually USB sticks, I won't suggest installing ventoy on an ssd. 
usually the ventoy installer will wipe the usb stick, create two partitions. a very small one which would contain the ventoy boot files and the other one where you can store your data and *ISO* Files.
In short, You just need to flash your usb stick with ventoy and it will create a drive named ventoy, you can put ISO files into it and boot from them directly through the BIOS boot menu.

for the windows profile on any computer, that's another complicated story (its called Windows-To-Go or something like that)

**Comment by alastair1970** [comment]
Thanks for your reply.  I think I’ve been led down the wrong rabbit hole.  I want something like Windows to go.

**Comment by steve6375** [comment]
Rufus will install WInToGo from a Windows ISO.

**Comment by wiz64** [comment]
> Thanks for your reply. I think I’ve been led down the wrong rabbit hole. I want something like Windows to go.

Welcome ! you can close this issue.
